src/Files.App/Helpers/Win32/Win32Helper.Storage.cs

```diff
@@ -935,10 +943,72 @@ public static bool UnsetFileAttribute(string lpFileName, FileAttributes dwAttrs)
 			return Win32PInvoke.SetFileAttributesFromApp(lpFileName, lpFileInfo.dwFileAttributes & ~dwAttrs);
 		}
 
+		public static unsafe bool CanCompressContent(string path)
+		{
+			path = Path.GetPathRoot(path) ?? string.Empty;
+
+			// MAX_PATH = 260 + 1 (null termination)
```

Comment:
`MAX_PATH` is 260 including null termination char (`\0`), and 3 (`C:\`) + 256 (= 259) can be used as path chars.

```c
// Definition
#define MAX_PATH 260

// Usage ("" of C/C++/C# appends \0 automatically and if you use all 260, this'd stack-overflow)
char szPath[MAX_PATH] = "C:\\Windows";
```

```suggestion
			// MAX_PATH = 260 (includes null termination char)
```

Reply:
Good to know, thanks!